generator/template.go

```diff
@@ -70,10 +71,10 @@ func (td *TemplateData) genFieldsColumnAddresses(buf *bytes.Buffer, fields []*Fi
 				buf.WriteString(fmt.Sprintf("return (*%s)(%s), nil\n", td.IdentifierType(f), f.fieldVarAddress()))
 			} else {
 				// can't scan a json if is nil
-				if f.IsJSON && f.IsPtr {
+				if (f.IsJSON || f.Kind == Interface) && f.IsPtr {
 					buf.WriteString(fmt.Sprintf(initNilPtrTpl, f.Name, f.Name, td.GenTypeName(f)))
```

Comment:
yes, it does https://github.com/src-d/go-kallax/pull/101/files#diff-35b32f9431b138f0c3a4d7b2dc2c8fc7R117